buildtools/src/main/java/org/apache/pulsar/tests/MockitoThreadLocalStateCleaner.java

```diff
@@ -72,16 +74,33 @@ public void cleanup() {
                         LOG.warn("Invalid usage of Mockito detected on thread {}."
                                         + " There is ongoing stubbing on mock of class={} instance={}",
                                 thread, mock.getClass().getName(), mock);
+                        try {
+                            clearInvocations(thread, mock);
+                        } catch (Exception e) {
+                            LOG.warn("Clearing invocations failed", e);
+                        }
```

Comment:
What's the difference between this private method and:

```java
import static org.mockito.Mockito.clearInvocations;

                        try {
                            clearInvocations(thread, mock);
                        } catch (Exception e) {
                            LOG.warn("Clearing invocations failed", e);
                        }
```

I may tend to avoid importing something from `internal` package.

Reply:
This is not something that is used in developer written tests. The solution will log warnings when tests aren't doing the right thing. Tests should call `Mockito.reset` for mocks.